### ISSUE TITLE
Visualization while training

### ISSUE THREAD

**Issue by at82** [LeapLabTHU/ActiveNeRF#4]
Hi, can I visualize the scene while training? Thanks!

**Comment by Panxuran** [comment]
Yes. You can set args.i_testset or args.i_video with a proper value to enable visualization in the training process.

**Comment by at82** [comment]
Thanks!
